src/AdminConsole/Components/Pages/App/Onboarding/GetStarted.razor.cs

```diff
@@ -0,0 +1,42 @@
+using Passwordless.AdminConsole.Components.Shared.Cards;
+using Passwordless.AdminConsole.Components.Shared.Icons.DeveloperIcons;
+using Passwordless.Common.Configuration;
+
+namespace Passwordless.AdminConsole.Components.Pages.App.Onboarding;
+
+public partial class GetStarted : BaseApplicationPage
+{
+    public IEnumerable<LinkCard.LinkCardModel> BackendCards => new LinkCard.LinkCardModel[]
+    {
+        new("REST API", "Use our simple REST API", typeof(OpenApiIcon), new Uri(Configuration.IsSelfHosted() ? $"{PasswordlessOptions.Value.ApiUrl}/swagger/index.html" : "https://docs.passwordless.dev/guide/api-documentation.html")),
+        new(".NET", "Docs + Examples", typeof(DotNetIcon), new Uri("https://docs.passwordless.dev/guide/backend/dotnet.html")),
+        new("Java", "Docs + Examples", typeof(JavaIcon), new Uri("https://docs.passwordless.dev/guide/backend/java.html")),
+        new("Node.js", "Docs + Examples", typeof(NodeJsIcon), new Uri("https://docs.passwordless.dev/guide/backend/nodejs.html")),
+        new("Python", "Docs + Examples", typeof(PythonIcon), new Uri("https://docs.passwordless.dev/guide/backend/python3.html"))
+    };
+
+    public IEnumerable<LinkCard.LinkCardModel> FrontendCards => new LinkCard.LinkCardModel[]
+    {
+        new("Android", "Docs + Examples", typeof(AndroidIcon), new Uri("https://docs.passwordless.dev/guide/frontend/android.html")),
+        new("ASP.NET Identity", "Docs + Examples", typeof(DotNetIcon), new Uri("https://docs.passwordless.dev/guide/frontend/aspnet.html")),
+        new("Javascript", "Docs + Examples", typeof(JavascriptIcon), new Uri("https://docs.passwordless.dev/guide/frontend/javascript.html")),
+        new("React", "Docs + Examples", typeof(ReactIcon), new Uri("https://docs.passwordless.dev/guide/frontend/react.html"))
+    };
```

Comment:
Avoid recreating the arrays every time the property is accessed:

```suggestion
    public IReadOnlyList<LinkCard.LinkCardModel> BackendCards { get; } = new LinkCard.LinkCardModel[]
    {
        new("REST API", "Use our simple REST API", typeof(OpenApiIcon), new Uri(Configuration.IsSelfHosted() ? $"{PasswordlessOptions.Value.ApiUrl}/swagger/index.html" : "https://docs.passwordless.dev/guide/api-documentation.html")),
        new(".NET", "Docs + Examples", typeof(DotNetIcon), new Uri("https://docs.passwordless.dev/guide/backend/dotnet.html")),
        new("Java", "Docs + Examples", typeof(JavaIcon), new Uri("https://docs.passwordless.dev/guide/backend/java.html")),
        new("Node.js", "Docs + Examples", typeof(NodeJsIcon), new Uri("https://docs.passwordless.dev/guide/backend/nodejs.html")),
        new("Python", "Docs + Examples", typeof(PythonIcon), new Uri("https://docs.passwordless.dev/guide/backend/python3.html"))
    };

    public IReadOnlyList<LinkCard.LinkCardModel> FrontendCards { get; } = new LinkCard.LinkCardModel[]
    {
        new("Android", "Docs + Examples", typeof(AndroidIcon), new Uri("https://docs.passwordless.dev/guide/frontend/android.html")),
        new("ASP.NET Identity", "Docs + Examples", typeof(DotNetIcon), new Uri("https://docs.passwordless.dev/guide/frontend/aspnet.html")),
        new("Javascript", "Docs + Examples", typeof(JavascriptIcon), new Uri("https://docs.passwordless.dev/guide/frontend/javascript.html")),
        new("React", "Docs + Examples", typeof(ReactIcon), new Uri("https://docs.passwordless.dev/guide/frontend/react.html"))
    };
```

Reply:
Configuration would be accessed from a static context, wouldn't be possible.